_pages/policies/tech-policies/security-incidents.md

```diff
@@ -12,19 +12,26 @@ See GSA’s Insite: [Report IT Security Incidents or Suspicious Activity](https:
 
 1. Do not click any links in the email. Do not delete it yet. You may mark it as spam.
 1. If you can, click the `Show Original` option in the "triangle" menu associated with the email. This will open a new window with the Original Message above and the raw text of the email below.
 1. Click on the `Download Original` link halfway down the page and it will save text of the email to your hard drive.
 1. Forward the email to itservicedesk@gsa.gov. As long as you haven't clicked on link or downloaded the file, you may stop here.
-1. If you suspect that the email has compromised your system with a download or link, you must also forward the email to gsa-ir@gsa.gov and devops@gsa.gov and attach the original text you downloaded. Please include *Security Incident* in the subject line, along with a brief description of the issue (Ex. Clicked on link in phishing email).
-1. Report the phishing email in the [#incident-response](https://gsa-tts.slack.com/messages/incident-response) Slack channel.
 1. After receiving your notification to GSA, IT will create a ticket and may contact you for more information. If you were unable to retrieve the raw headers above, IT may need to access your computer to view the email.
-1. Mark the email as a phishing email based on confirmation from the GSA IR team by selecting the `Report phishing` option in the same "triangle" menu associated with the email.
+1. After GSA IR confirms that they have all they need to investigate, mark the email as a phishing email by selecting the `Report phishing` option in the same "triangle" menu associated with the email.
 
-You might be tempted to simply mark phishing emails as spam and otherwise ignore them, but if you accidentally (or intentionally) click a link or receive a download from a suspect email, you must report it as an incident following the steps above. Successful phishing attacks are security incidents and should be reported immediately. Phishing emails that are automatically routed to your spam folder do not need to be reported. Your vigilance also helps GSA IT to prepare against similar phishing attacks that might be sent to other users.
+### Reporting phishing incidents
+If you also clicked on a link in a phishing email, follow these steps to report to GSA IR:
+
+1. If you haven't already, follow the steps above for reporting phishing emails, and then continue to the next step.
+1. Forward the phishing email to gsa-ir@gsa.gov and devops@gsa.gov and attach the original text you downloaded. Please include *Security Incident* in the subject line, along with a brief description of the issue (Ex. Clicked on link in phishing email).
+1. Report the phishing email in the [#incident-response](https://gsa-tts.slack.com/messages/incident-response) Slack channel.
+1. After GSA IR confirms that they have all they need to investigate, mark the email as a phishing email by selecting the `Report phishing` option in the same "triangle" menu associated with the email.
```

Comment:
It almost seems like the instructions can direct the reporter to click `Report Phishing` twice.  Maybe something like mark the email if it has not already been marked?

Reply:
Fixed in https://github.com/18F/handbook/pull/635/commits/183a5646e903f0f34540a729232875c0a39d0557 .